frontend/src/pages/RegistrationsPage.tsx

```diff
@@ -20,38 +22,30 @@ export default function RegistrationsPage() {
 
         event.preventDefault();
 
-
-
-        const newUser: UserRegisterData = {
-            "email": userEmail,
-            "name": userName,
-            "password": userPassword,
-
-        }
-
-        axios.post("/auth/register", newUser)
-            .then((reponse) => reponse.data)
-            .then(() => {
-                setUserEmail("");
-                setUserName("");
-                setUserPassword("");
-                setUserRepeatPassword("");
-                setErrorMessage("")
-                toast.success("Account Created!")
-                navigate("/");
-            })
+        register(userEmail, userName, userPassword, userRepeatPassword).then(() => {
+            if (userPassword !== userRepeatPassword)
+                toast.error('Passwords do not match')
+            setUserEmail("");
```

Comment:
Als Idee: 
Wenn password und repeatPassword nicht gleich sind, warum willst du auch email und username zu leeren String setzen? Soll user komplett von Anfang gehen?

Reply:
Danke für den Review. Eigentlich will ich gar nichts leeren. ich habe gedacht das was ab zeile 28 anfängt quasi else block wäre!??

Reply:
Wenn ich mich nicht irre, wenn man "if" ohne geschweifte Klammern nutzt, dann ist es "one-line-if" und alles was danach (nach erste Zeile) kommt,  ist schon für alle Falle implementiert werden würde. Bin aber kein Profi in javascript, wenn du {} nutzen würdest, würde es einfacher :-)